src/app/shared/collections-grid/collections-grid.component.html

```diff
@@ -1,29 +1,55 @@
 <div class="collections-grid">
   <ng-container *ngIf="!loading; else loadingTemplate">
     <div *ngIf="collections; else noneTemplate" class="collection-chooser">
-      <div *ngFor="let c of collections" class="collection-chooser__collection" [ngClass]="{'selected': currentCollection === c.abvName}" (click)="select(c.abvName)">
-        <div *ngIf="c.hasLogo; else genericCollectionImage" class="collection__logo" [ngStyle]="{backgroundImage: 'url(assets/images/collections/' + c.abvName + '.png)'}"></div>
-        <div class="collection__name">{{ c.name }}</div>
+      <div
+        *ngFor="let c of collections"
+        [tip]="
+          preselectReason || 'This collection cannot be selected right now.'
+        "
+        tipDisabled="{{ !isPreselected(c.abvName) }}"
+        tipLocation="above"
+      >
+        <div
+          class="collection-chooser__collection"
+          [ngClass]="{
+            selected: currentCollection === c.abvName,
+            'collection-chooser__collection--preselected': isPreselected(
+              c.abvName
+            )
+          }"
+          (click)="!isPreselected(c.abvName) && select(c.abvName)"
+        >
+          <div
+            *ngIf="c.hasLogo; else genericCollectionImage"
+            class="collection__logo"
+            [ngStyle]="{
+              backgroundImage:
+                'url(assets/images/collections/' + c.abvName + '.png)'
+            }"
+          ></div>
+          <div class="collection__name">{{ c.name }}</div>
+        </div>
       </div>
     </div>
   </ng-container>
-  
+
   <!-- loading -->
   <ng-template #loadingTemplate>
     <div class="loading-template">
       <span><i class="far fa-spinner-third fa-spin"></i></span>
       Loading collections...
     </div>
   </ng-template>
-  
+
   <ng-template #noneTemplate>
     <div class="none-template">
       <span><i class="far fa-times"></i></span>
-      Uh-oh! We weren't able to get a list of collections. Please try again later.
+      Uh-oh! We weren't able to get a list of collections. Please try again
```

Comment:
It might be a good idea to reword this error message. "Uh-oh!" seems a little sarcastic.

Reply:
@DonSeannelly since this is your copy, what are your thoughts? Personally I think I like it the way it is.

![image](https://user-images.githubusercontent.com/7431390/59365058-e2be8380-8d05-11e9-9af3-3721fc1e9e52.png)

Reply:
I think we need to spend some time on identifying the tone and voice we want to use in copy and user-facing messages on the system. We alternate between fairly cold and robotic to somewhat playful at times.

If the "Uh-oh" seems a little off-putting maybe we could just drop it.

Also, question on this now that I'm looking at it again. Why did we remove `later .`?

Reply:
We didn't, later is just pushed down onto the next line for some reason

Reply:
Oh, and we absolutely need a voice-and-tone guideline set. Will be part of the forever impending style guide I keep promising.

Reply:
@nrwinner maybe we could break it out of that so the can doesn't get kicked too far down the road. It seems to be something that is coming up on a weekly basis. I'd like to get even just a preliminary draft put together in the next few weeks.

Reply:
I've removed the 'Uh-oh' for the time being until we solidify our voice and tone guidelines